COVID19_Fallzahlen_CH_total.csv

```diff
@@ -139,25 +139,50 @@ date,time,abbreviation_canton_and_fl,ncumul_tested,ncumul_conf,ncumul_hosp,ncumu
+2020-03-14,,VS,,76,17,1,1,,1,https://www.vs.ch/documents/6756452/7008787/Sit%20Epid%20-%20%C3%89tat%20Stand%2025.03.2020.pdf
+2020-03-15,,VS,,98,22,1,1,,1,https://www.vs.ch/documents/6756452/7008787/Sit%20Epid%20-%20%C3%89tat%20Stand%2025.03.2020.pdf
+2020-03-16,,VS,,116,24,1,1,,2,https://www.vs.ch/documents/6756452/7008787/Sit%20Epid%20-%20%C3%89tat%20Stand%2025.03.2020.pdf
+2020-03-17,,VS,,173,29,2,1,,3,https://www.vs.ch/de/web/coronavirus/info?p_p_id=com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_1rjMHS5sCcaN&p_p_lifecycle=0&p_p_state=normal&p_p_mode=view&_com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_1rjMHS5sCcaN_delta=5&p_r_p_resetCur=false&_com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_1rjMHS5sCcaN_cur=2#collapse6927494
+2020-03-18,,VS,,225,33,2,2,,3,https://www.vs.ch/documents/6756452/7008787/Sit%20Epid%20-%20%C3%89tat%20Stand%2025.03.2020.pdf
+2020-03-19,,VS,,311,42,2,2,,4,https://www.vs.ch/de/web/coronavirus#collapse6955818
+2020-03-20,,VS,,346,47,5,5,,6,https://www.vs.ch/de/web/coronavirus
+2020-03-21,,VS,,433,55,6,5,,7,https://www.vs.ch/de/web/coronavirus
+2020-03-22,,VS,,490,64,7,6,,10,https://www.vs.ch/de/web/coronavirus
+2020-03-23,,VS,,527,70,10,7,,12,https://www.vs.ch/de/web/coronavirus
+2020-03-24,16:00,VS,,606,80,11,9,,13,https://www.vs.ch/de/web/coronavirus
+2020-03-25,15:00,VS,,651,84,11,9,,14,https://www.vs.ch/documents/6756452/7008787/Sit%20Epid%20-%20%C3%89tat%20Stand%2025.03.2020.pdf
+2020-03-03,"",JU,"",1,"","","","","",https://www.jura.ch/fr/Autorites/Coronavirus/Chiffres/Evolution-des-cas-COVID-19-dans-le-Jura.html
+2020-03-04,"",JU,"",1,"","","","","",https://www.jura.ch/fr/Autorites/Coronavirus/Chiffres/Evolution-des-cas-COVID-19-dans-le-Jura.html
+2020-03-05,"",JU,"",2,"","","","","",https://www.jura.ch/fr/Autorites/Coronavirus/Chiffres/Evolution-des-cas-COVID-19-dans-le-Jura.html
+2020-03-06,"",JU,"",3,"","","","","",https://www.jura.ch/fr/Autorites/Coronavirus/Chiffres/Evolution-des-cas-COVID-19-dans-le-Jura.html
+2020-03-07,"",JU,"",4,"","","","","",https://www.jura.ch/fr/Autorites/Coronavirus/Chiffres/Evolution-des-cas-COVID-19-dans-le-Jura.html
+2020-03-08,"",JU,"",4,"","","","","",https://www.jura.ch/fr/Autorites/Coronavirus/Chiffres/Evolution-des-cas-COVID-19-dans-le-Jura.html
+2020-03-09,"",JU,"",5,"","","","","",https://www.jura.ch/fr/Autorites/Coronavirus/Chiffres/Evolution-des-cas-COVID-19-dans-le-Jura.html
+2020-03-10,"",JU,"",6,"","","","","",https://www.jura.ch/fr/Autorites/Coronavirus/Chiffres/Evolution-des-cas-COVID-19-dans-le-Jura.html
+2020-03-11,"",JU,"",6,"","","","","",https://www.jura.ch/fr/Autorites/Coronavirus/Chiffres/Evolution-des-cas-COVID-19-dans-le-Jura.html
```

Comment:
Jura reports 0 for three dates (4.3, 8,3, 11.3). This is a bit weird, as this makes no sense when reporting cumulative figures. The value for the 25.3 is very close to that of the BAG, so it truly seems to be a cumulative number and not something else (like the  number of currently positive). Just to ensure everything is fine here we could ask, what these 0 mean?